modules/zipkin-http-exporter/src/main/scala/io/janstenpickle/trace4cats/zipkin/ZipkinHttpSpanExporter.scala

```diff
@@ -1,32 +1,24 @@
 package io.janstenpickle.trace4cats.zipkin
 
 import cats.Foldable
-import cats.effect.kernel.{Async, Resource}
+import cats.effect.kernel.Async
+import cats.syntax.functor._
 import io.janstenpickle.trace4cats.`export`.HttpSpanExporter
 import io.janstenpickle.trace4cats.kernel.SpanExporter
+import org.http4s.Uri
 import org.http4s.client.Client
-import org.http4s.blaze.client.BlazeClientBuilder
-
-import scala.concurrent.ExecutionContext
 
 object ZipkinHttpSpanExporter {
 
-  def blazeClient[F[_]: Async, G[_]: Foldable](
-    host: String = "localhost",
-    port: Int = 9411,
-    ec: Option[ExecutionContext] = None
-  ): Resource[F, SpanExporter[F, G]] = for {
-    client <- ec.fold(BlazeClientBuilder[F])(BlazeClientBuilder[F].withExecutionContext).resource
-    exporter <- Resource.eval(apply[F, G](client, host, port))
-  } yield exporter
-
   def apply[F[_]: Async, G[_]: Foldable](
     client: Client[F],
     host: String = "localhost",
     port: Int = 9411
   ): F[SpanExporter[F, G]] =
-    HttpSpanExporter[F, G, String](client, s"http://$host:$port/api/v2/spans", ZipkinSpan.toJsonString[G](_))
+    Async[F].fromEither(Uri.fromString(s"http://$host:$port/api/v2/spans")).map { uri =>
```

Comment:
I'd prefer to use `liftTo` here if that's OK? so `Async[F].fromEither(Uri.fromString(s"http://$host:$port/api/v2/spans"))` becomes `Uri.fromString(s"http://$host:$port/api/v2/spans").liftTo[F]` with an import of `cats.syntax.either._`